locales/cs.json

```diff
@@ -0,0 +1,193 @@
+  "LaTeX Inline Open Delimiter": "inline otevírací oddělovač v LaTeX",
+  "LaTeX Inline Close Delimiter": "inline uzavírací oddělovač v LaTeX",
+  "LaTeX Block Open Delimiter": "otevírací oddělovač bloku v LaTeX",
+  "LaTeX Block Close Delimiter": "uzavírací oddělovač bloku v LaTeX",
+  "PlantUML Server": "PlantUML server",
+  "Community": "Komunita",
+  "Subscribe to Newsletter": "Přihlásit se k odběru novinek",
+  "GitHub": "GitHub",
+  "Blog": "Blog",
+  "Facebook Group": "Facebook Skupina",
+  "Twitter": "Twitter",
+  "About": "O aplikaci",
+  "Boostnote": "Boostnote",
+  "An open source note-taking app made for programmers just like you.": "Open source aplikace pro tvorbu poznámek vytvořená programátory jako jste vy.",
+  "Website": "Webová stránka",
+  "Development": "Vývoj",
+  " : Development configurations for Boostnote.": " : Vývojové konfigurace pro Boostnote.",
+  "Copyright (C) 2017 - 2019 BoostIO": "Copyright (C) 2017 - 2019 BoostIO",
+  "License: GPL v3": "License: GPL v3",
+  "Analytics": "Analytika",
+  "Boostnote collects anonymous data for the sole purpose of improving the application, and strictly does not collect any personal information such the contents of your notes.": "Boostnote sbírá anonymní data pouze pro účely zlepšování aplikace, a v žádném případě nesbírá žádné osobní údaje, jako je například obsah vašich poznámek.",
+  "You can see how it works on ": "Můžete se podívat, jak to funguje ",
+  "You can choose to enable or disable this option.": "Tuto možnost můžete povolit nebo zakázat.",
+  "Enable analytics to help improve Boostnote": "Povolit analytiku pro další zlepšování Boostnote",
+  "Crowdfunding": "Crowdfunding",
+  "Dear Boostnote users,": "Vážení uživatelé Boostnote,",
+  "Thank you for using Boostnote!": "Děkujeme, že používáte Boostnote!",
+  "Boostnote is used in about 200 different countries and regions by an awesome community of developers.": "Boostnote je používán v přibližně 200 různých zemích a regionech úžasnou komunitou vývojářů.",
+  "To support our growing userbase, and satisfy community expectations,": "Abychom podpořili rostoucí uživatelskou základnu a naplnili očekávání komunity,",
+  "we would like to invest more time and resources in this project.": "rádi bychom do tohoto projektu investovali více času a zdrojů.",
+  "If you use Boostnote and see its potential, help us out by supporting the project on OpenCollective!": "Pokud používáte Boostnote a vidíte jeho potenciál, pomozte nám podporou projektu na OpenCollective!",
+  "Thanks,": "Děkujeme,",
+  "The Boostnote Team": "Tým Boostnote",
+  "Support via OpenCollective": "Podpořit přes OpenCollective",
+  "Language": "Jazyk",
+  "Default New Note": "Výchozí nová poznámka",
+  "English": "Angličtina",
+  "German": "Němčina",
+  "French": "Francouzština",
+  "Show \"Saved to Clipboard\" notification when copying": "Zobrazit při kopírování upozornění \"Uloženo do schránky\"",
+  "All Notes": "Všechny poznámky",
+  "Starred": "Starred",
```

Comment:
Is `Starred` similar to `Favourites` in meaning? If so, the translation could be `Oblíbené` (direct translation of `Favourites`).